packages/block-directory/src/plugins/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+
+/**
+ * Internal dependencies
+ */
+import InserterMenuDownloadableBlocksPanel from './inserter-menu-downloadable-blocks-panel';
+
+registerPlugin( 'block-directory', {
+	render() {
+		return <InserterMenuDownloadableBlocksPanel />;
```

Comment:
Don't you need a new slot or something? or was it already there?

Reply:
It already uses an experimental slot, though @noisysocks had some ideas about moving the implementation to `edit-post` instead.